_disability-benefits/learn/eligibility/claim-types.md

```diff
@@ -1,71 +0,0 @@
----
-title: Claim types
```

Comment:
Why does it look like all the Claim Types pages are going away? I'm assuming I'm misunderstanding what's happening here, but just checking to make sure these pages aren't being deleted by accident.